### PR TITLE
Page Playlists/artists, unite profile menu and leftside menu in one

### PR DESCRIPTION
**Pull request by Slawisch**

### CODE REVIEW

File: backend/Perflow/Services/ArtistReactionService.cs [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+                await context.ArtistReactions.AddAsync(artistReaction);
+
+                await context.SaveChangesAsync();
+            }
+            else
+            {
+                throw new ArgumentException("Reaction already exists");
+            }
+        }
```
Comment (proskurnicha) [inline backend/Perflow/Services/ArtistReactionService.cs:52]: Use Return Early Pattern
https://medium.com/swlh/return-early-pattern-3d18a41bba8

File: backend/Perflow/Services/ArtistReactionService.cs [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+            {
+                context.ArtistReactions.RemoveRange(reactions);
+
+                await context.SaveChangesAsync();
+            }
+            else
+            {
+                throw new ArgumentException("Reaction does not exist");
+            }
+        }
```
Comment (proskurnicha) [inline backend/Perflow/Services/ArtistReactionService.cs:72]: And here

**Review by proskurnicha — CHANGES_REQUESTED** [review]

File: backend/Perflow/Common/DTO/Users/UserReadDTO.cs [hunk 3]
```diff
@@ -11,6 +11,11 @@ public class UserReadDTO
         public string Email { get; set; }
         public DateTimeOffset Birthday { get; set; }
         public bool Gender { get; set; }
+        public string Description { get; set; }
+        public string Country { get; set; }
+        public string Password { get; set; }
+        public string Salt { get; set; }
```
Comment (NikitaKrasnovSW) [inline backend/Perflow/Common/DTO/Users/UserReadDTO.cs:17]: Why do you use Salt and Password in dto?

File: backend/Perflow/DataAccess/Context/ModelBuilderExtensions.cs [hunk 4]
```diff
@@ -25,6 +26,7 @@ public static void Configure(this ModelBuilder modelBuilder)
             modelBuilder.ApplyConfiguration(new UserSettingsConfiguration());
             modelBuilder.ApplyConfiguration(new NotificationConfiguration());
             modelBuilder.ApplyConfiguration(new ArtistFollowerConfiguration());
+            Seed(modelBuilder);
```
Comment (NikitaKrasnovSW) [inline backend/Perflow/DataAccess/Context/ModelBuilderExtensions.cs:29]: We no longer have seeding automated

File: backend/Perflow/DataAccess/Context/ModelBuilderExtensions.cs [hunk 5]
```diff
@@ -55,6 +59,7 @@ public static void Seed(this ModelBuilder modelBuilder)
                     new { UsersId = users[3].Id, RolesId = roles[0].Id }
                     )
                 );
+
```
Comment (NikitaKrasnovSW) [inline backend/Perflow/DataAccess/Context/ModelBuilderExtensions.cs:62]: Delete empty line

File: backend/Perflow/Migrations/20210803162535_RunSeeding.cs [hunk 6]
```diff
@@ -0,0 +1,268 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace Perflow.Migrations
+{
+    public partial class RunSeeding : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
```
Comment (NikitaKrasnovSW) [inline backend/Perflow/Migrations/20210803162535_RunSeeding.cs:8]: Don't add data modifications in migrations. Migrations should only change data structure. This approach is not safe for production environment

File: backend/Perflow/Migrations/20210803170353_ArtistReactionsSeed.cs [hunk 7]
```diff
@@ -0,0 +1,33 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace Perflow.Migrations
+{
+    public partial class ArtistReactionsSeed : Migration
```
Comment (NikitaKrasnovSW) [inline backend/Perflow/Migrations/20210803170353_ArtistReactionsSeed.cs:5]: Delete seeding migration

File: frontend/src/app/typings.d.ts [hunk 8]
```diff
@@ -1,4 +1,5 @@
-declare let process:Process;
+/* eslint-disable no-var */
+declare var process:Process;
```
Comment (NikitaKrasnovSW) [inline frontend/src/app/typings.d.ts:2]: Add space

File: frontend/src/app/typings.d.ts [hunk 9]
```diff
@@ -1,4 +1,5 @@
...
 
 interface Process{
   env:Env
```
Comment (NikitaKrasnovSW) [inline frontend/src/app/typings.d.ts:5]: Fix formatting

File: frontend/src/styles.sass [hunk 10]
```diff
@@ -24,8 +24,7 @@ $input-background-color: #282F37
 $drk-rain: #282F37
 $orange-gradient: linear-gradient(90deg, $light-orange 0%, $medium-orange 100%)
 $reversed-orange-gradient: linear-gradient(90deg, $medium-orange 0%, $light-orange 100%)
-$text: #FAFBFF
-$orange-gradient: linear-gradient(90deg, $light-orange 0%, $medium-orange 100%)
+$text:#FAFBFF
```
Comment (NikitaKrasnovSW) [inline frontend/src/styles.sass:27]: Add space

**Review by NikitaKrasnovSW — CHANGES_REQUESTED** [review]

File: backend/Perflow/Common/DTO/Users/UserReadDTO.cs (hunk 3 above)
Comment (Slawisch) [reply backend/Perflow/Common/DTO/Users/UserReadDTO.cs:17]: It isn't my class, I've just changed existing records to class.

**Comment by Slawisch** [comment]
Changes from this branch are in 'albums-list'. To my mind, this branch is supposed to be deleted.

**Pull request closed without merging**